projects/lendai-us/design-system/src/lib/phone-input/phone-input.component.ts

```diff
@@ -114,20 +124,39 @@ export class PhoneInputComponent
     this.sub$.unsubscribe();
   }
 
-  writeValue(value: string): void {
-    const number = parseNumber(value ?? '');
+  applySearchFilter(searchText: string): void {
+    this.filteredList = this.countries.filter(item => {
+      if (
+        item.name.toString().toLowerCase().indexOf(searchText.toLowerCase()) !==
```

Comment:
split the `item.name` modification and condition expression into separate lines to increase readability, `searchString` can be lower cased only once outside of the loop, return statement can be simplified e.g. 

```typescript
searchText = searchText.toLowerCase();
this.filteredList = this.countries.filter(item => {
  const name = item.name.toString().toLowerCase();
  return name.indexOf(searchText) !== -1;
});
...
```